content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -0,0 +1,310 @@
+# Build an app with RSFormView in Swift
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team and recently has started to write about some of the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
+
+Note: this post is meant to help you integrate RSFormView in your app, if you are interested in the motivation and implementation of RSFormView please take a look at the previous article [Link to previous article].
+
+## Creating the project   
+
+ During this article we will be implementing step by step an single view iOS application with RSFormView integrated.
```

Comment:
step-by-step

Reply:
Restructure sentence: "During this article we will be implementing a single view iOS application with RSFormView integrated, step-by-step."

Or: "This article is a step-by-step guide on how to implement a single view iOS application with RSFormView integrated"